src/test/java/no/unit/nva/database/RoleDbTest.java

```diff
@@ -32,50 +35,51 @@ public RoleDbTest() throws InvalidRoleException {
     }
 
     @BeforeEach
-    public void init() {
+    void init() {
         initializeTestDatabase();
         mapper = new DynamoDBMapper(localDynamo);
     }
 
     @Test
-    public void roleDbHasBuilder() {
-        assertDoesNotThrow(RoleDb::newBuilder);
+    void roleDbHasBuilder() {
+        RoleDb.Builder builder = RoleDb.newBuilder();
+        assertNotNull(builder);
     }
 
     @Test
-    public void roleDbHasRoleName() throws InvalidRoleException {
+    void roleDbHasRoleName() throws InvalidRoleException {
```

Comment:
When public, checkstyle demands documentation from parameterized tests. Therfore, for homogeneity, i set all tests to default access level.